lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -229,6 +229,18 @@ int main(int argc, char* argv[]) {
     policy_loader.stop();
   });
 
+  //Start off a thread to periodically poll stats from Pipelined
+  auto rule_store = std::make_shared<magma::StaticRuleStore>();
+  magma::PollStats periodic_stats_requester;
+  std::thread periodic_stats_requester_thread([&]() {
+    periodic_stats_requester.start_loop(
+      [&](std::vector<magma::PollStats> request){
+        //call enforcer function
+      },
+      config["rule_update_inteval_sec"].as<uint32_t>());
+      periodic_stats_requester.stop();
+  });
```

Comment:
why is this on the main. Wouldn't it be better if this were ran from localEnforcer? This way in the future we may be able to control the poll interval dynamically.

Or Maybe you can create a `run` method on  to start this on localEnforcer